src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+function formatNumber(num) {
+  return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+}
```

Comment:
The `formatNumber` function is correctly implemented to format numbers with commas as thousand separators. No changes are needed here.